nvflare/lighter/provision.py

```diff
@@ -25,6 +25,30 @@
 from nvflare.lighter.spec import Participant, Project, Provisioner
 from nvflare.lighter.utils import load_yaml
 
+adding_client_error_msg = """
+name: $SITE-NAME
+type: client
+org: $ORGANIZATION_NAME
+enable_byoc: $BOOLEAN
+components:
+    resource_manager:    # This id is reserved by system.  Do not change it.
+        path: nvflare.app_common.resource_managers.list_resource_manager.ListResourceManager
+        args:
+        resources:
+            gpu: [0, 1, 2, 3]
+    resource_consumer:    # This id is reserved by system.  Do not change it.
+        path: nvflare.app_common.resource_consumers.gpu_resource_consumer.GPUResourceConsumer
+        args:
+        gpu_resource_key: gpu
+"""
+
+adding_user_error_msg = """
+name: $USER_EMAIL_ADDRESS
+org: $ORGANIZATION_NAME
+roles:
+    - $ROLE
+"""
```

Comment:
I am thinking if we put these "example yaml" in separate files.
Then in the future if we are going to modify the content we don't need to change the code in this file.

Reply:
My thought was that the info for clients/users were still undecided yet.  After everything is finalized, this user-facing msg can be in a separate file.